src/main/java/kenigsberg/wordledictionary/WordleGameFrame.java

```diff
@@ -4,23 +4,228 @@
+        JPanel flowLayout1 = new JPanel();
+        flowLayout1.setLayout(new FlowLayout());
+
+        JPanel flowLayout2 = new JPanel();
+        flowLayout2.setLayout(new FlowLayout());
+
+        JPanel flowLayout3 = new JPanel();
+        flowLayout3.setLayout(new FlowLayout());
+
+
+        JPanel southPanel = new JPanel();
+        southPanel.setLayout(new BoxLayout(southPanel, BoxLayout.Y_AXIS));
+
+        southPanel.add(flowLayout1);
+        southPanel.add(flowLayout2);
+        southPanel.add(flowLayout3);
+
+        JLabel title = new JLabel("Welcome to Wordle Game");
+        mainPanel.add(title, BorderLayout.NORTH);
+        title.setHorizontalAlignment(JLabel.CENTER);
+
+        mainPanel.add(centerPanel, BorderLayout.CENTER);
+        mainPanel.add(southPanel, BorderLayout.SOUTH);
+
+
+        //Add code here
+
+
+        //For every iteration of JLabel 2D Array, create Label, add to array, and add to centerPanel
+        for (int i = 0; i < fields.length; i++) {
+            for (int j = 0; j < fields[i].length; j++) {
+                fields[i][j] = new JLabel();
+                fields[i][j].setBorder(BorderFactory.createLineBorder(Color.black));
+                centerPanel.add(fields[i][j]);
+            }
+        }
+
+        //For every Button of keyboard, want to create new button and set the letter to the button
+        // (in order of how it will appear in the keyboard)
+        for (int i = 0; i < keyboard.length; i++) {
+            keyboard[i] = new JButton(letters[i]);
+            keyboard[i].setFont(new Font(Font.MONOSPACED, Font.BOLD, 15));
+        }
+
+        //add first line of keyboard to flowLayout1
+        for (int i = 0; i < 10; i++) {
+            flowLayout1.add(keyboard[i]);
+        }
+
+        //add second line keyboard to flowLayout2
+        for (int i = 10; i < 19; i++) {
+            flowLayout2.add(keyboard[i]);
+        }
+
+        //add third line keyboard to flowLayout3
+
+        enter = new JButton("Enter");
+        enter.setFont(new Font(Font.MONOSPACED, Font.BOLD, 15));
+        flowLayout3.add(enter);
+
+        for (int i = 19; i < 26; i++) {
+            flowLayout3.add(keyboard[i]);
+        }
+        backspace = new JButton("Del");
+        backspace.setFont(new Font(Font.MONOSPACED, Font.BOLD, 15));
+        flowLayout3.add(backspace);
+
+
+        //AT THIS POINT THE UI IS CREATED
+
+
+        //Now want to listen for the letters being pressed
+        //(BOTH by pressing buttons & listening to keyboard typed)
+
+
+        //This will listen out for buttons pressed
+
+        //Letters pressed
+        for (JButton letter : keyboard) {
+
+            letter.addActionListener(e -> {
+                //Now if one of the letters have been pressed
+                // (on screen buttons) want to call the Controller
+                controller.addLetter(letter.getText());
+                System.out.println("Letter pressed");
```

Comment:
Don't rely on putting printlns everywhere.

Reply:
I put them in before I filled the methods in Controller so that I can check that my program is picking up on the keyboard (clicked & typed)

I will remove them later if that's alright. Still want them around to debug